src/odemis/dataio/tiff.py

```diff
@@ -207,20 +208,20 @@ def _readTiffTag(tfile):
     # informative metadata
     val = tfile.GetField(T.TIFFTAG_PAGENAME)
     if val is not None:
-        md[model.MD_DESCRIPTION] = val
+        md[model.MD_DESCRIPTION] = val.decode('utf-8')
```

Comment:
For all the `decode()`, please add a "ignore" error handler, so that if the file metadata is a little bit unexpected, we don't entirely fail to read the file.

Reply:
Should I also add "ignore" to the encode statements or do we actually want it to fail in this case (e.g. because we don't support Chinese characters in the description)?